usagov/vote-gov-drupal
# VOTE-3014 Update NVRF to latest and update entry file names

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-3014

## Description

Update NVRF to v0.19.0

## Deployment and testing

### Post-deploy steps

1. cd into vote_nvrf and run `npm i`
2. run `lando drush cr`

### QA/Testing instructions

1. visit http://vote-gov.lndo.site/register/arizona/mail-in-form-filler and confirm the NVRF app works and generates a filled PDF
2. visit http://vote-gov.lndo.site/es/registrar/arizona/llenador-de-formulario-por-correo-postal and confirm it works in spanish as well

## Checklist for the Developer




## Checklist for the Peer Reviewers
